gent_base.theme

```diff
@@ -733,3 +734,56 @@ function gent_base_preprocess_pager(&$variables) {
   $variables['items']['first']['text'] = '1';
   $variables['items']['last']['text'] = $total;
 }
+
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function gent_base_preprocess_paragraph__call_to_action(&$variables) {
+  $variables['primary_document_size'] = format_size(_gent_base_get_entity_reference_file_size($variables, 'field_primary_action_document'));
```

Comment:
How will this be displayed if the `_gent_base_get_entity_reference_file_size()` function returns NULL?

Reply:
Twig gaat een check doen op de variables die nodig zijn.